ClassicPress-plugins/classic-commerce
# Change sample data files to show CC products, remove xml file

### All Submissions:






### Changes proposed in this Pull Request:

Changes the content of the sample_products.csv file to reflect cc-based products. Changes the image URLs to pick up files from a downloads folder on the CC site. Removes the xml file completely. Docs will be revised to only show method for using csv through the Classic Commerce product import process.

Closes #272

### How to test the changes in this Pull Request:

1. Download sample_products.csv
2. On a clean test site run the Classic Commerce product import process
3. Check the products uploaded, especially images.
4. A list of imported products and products types is available here for checking: https://classiccommerce.cc/docs/installation-and-setup/importing-sample-data/

### Screenshot - before:



### Screenshot - after:



### Other information:
